docs/readthedocs/source/doc/LLM/Quickstart/install_linux_driver.md

```diff
@@ -0,0 +1,82 @@
+# Install GPU Driver on Linux
```

Comment:
No need to downgrade kernel now. @qiuxin2012 - have we updated the doc?

Reply:
The lastest ubuntu 22.04's kernel is upgraded to 6.5. From Fred and Kai's feed back, the kernel 6.5 has some performance issue. If user's kernel is 6.2(not the latest ubuntu 22.04), we don't need to downgrade kernel.

Reply:
1. Remove downgrade kernel in quick start
2. What's the perf gap of 6.2 vs. 6.5?
3. Is there a Ubuntu version (e.g., 22.04.3?) that uses Kernel 6.2?
4. We can add a note on the 6.5 kernel performance issue, and refer the user to the installation for more details

Reply:
https://github.com/analytics-zoo/nano/issues/964#issuecomment-1956444582 some previous results for kernel 6.2 vs 6.5. The gap is not large, but setting `export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1` significantly increases the latency..

Reply:
This PR first adds driver installation commands for kernel 5.19; 6.5 will be tested and added in a following PR.

Reply:
sure.

Reply:
Ubuntu version is not binding to a special linux kernel version. Ubuntu 22.04.3 can install 5.19, 6.2, 6.5 kernel at the same time. User can choose one of them at the booting time.
And when installing a new system, if the machine is connected to internet, ubuntu will download the updates and install the latest kernel. 
So the requirement(ubuntu 22.04.3) can't guarantee that user's kernel is 6.2.

Reply:
Most user don't know the relations of the kernel and ubuntu version. When I was a beginner of Ubuntu, I always choosed yes when software market pushed updates to me. One day, I found I got a five different linux kernels.